<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>264</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImgPrev">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>select img 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImgCurr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>select img 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_img1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_img2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>194</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>smoothing 
sigma</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="smoothing_sigma">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>194</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>window 
size</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="window_size">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>194</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>254</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>smoothing 
window 
size</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="smoothing_window_size">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>264</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>max
 intensity 
error</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="max_error">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>264</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>max
scaling
level</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="max_scaling_level">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>324</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectSaveFile">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>save to file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_flow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
